### PR TITLE
Add support for Card Attachments(Adaptive Card)

### PR DESCRIPTION
**Pull request by kishorepmr**
# Pull Request Template

## Description

- Updated comments to support cards
- Added decryptPropCardItem to decrypt singleCard without altering array order

## Type of Change

Please delete options that are not relevant.

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

## Test Coverage

Please describe the tests that you ran to verify your changes. Provide instructions so we can reproduce. Please also list any relevant details for your test configuration

- [ ] Test A
- [ ] Test B

**Test Configuration**:
* SDK Version
* Node/Browser Version
* NPM Version

## Checklist:

- [X] My code follows the style guidelines of this project
- [X] I have performed a self-review of my own code
- [X] I have commented my code, particularly in hard-to-understand areas
- [X] I have made corresponding changes to the documentation
- [X] My changes generate no new warnings
- [X] I have added tests that prove my fix is effective or that my feature works
- [X] New and existing unit tests pass locally with my changes
- [X] Any dependent changes have been merged and published in downstream modules
@mattnorris @taymoork2 @adamweeks

### CODE REVIEW

File: packages/node_modules/@ciscospark/internal-plugin-conversation/test/unit/spec/decrypt-transforms.js [hunk 1]
```diff
@@ -27,6 +27,25 @@ describe('plugin-conversation', () => {
       });
     });
 
+    describe('decryptComment()', () => {
+      it('calls the correct method if a array prop is passed to it', () => {
```
Comment (mattnorris) [inline packages/node_modules/@ciscospark/internal-plugin-conversation/test/unit/spec/decrypt-transforms.js]: 1. ...`an array`...
2. Will there be other props tested in the future?

File: packages/node_modules/@ciscospark/internal-plugin-conversation/test/unit/spec/decrypt-transforms.js [hunk 2]
```diff
@@ -27,6 +27,25 @@ describe('plugin-conversation', () => {
...
+        };
+
+        transform.fn(ctx, key, activity);
+        assert.equal(transformStub.lastCall.args[0], 'decryptPropCardItem');
```
Comment (mattnorris) [inline packages/node_modules/@ciscospark/internal-plugin-conversation/test/unit/spec/decrypt-transforms.js]: @taymoork2 should Kishore use an `expect` from `chai` here instead?

File: packages/node_modules/@ciscospark/internal-plugin-conversation/test/unit/spec/decrypt-transforms.js (hunk 1 above)
Comment (kishorepmr) [reply packages/node_modules/@ciscospark/internal-plugin-conversation/test/unit/spec/decrypt-transforms.js]: No Matt there will not be any other props other than array for cards.

File: packages/node_modules/@ciscospark/internal-plugin-conversation/test/unit/spec/decrypt-transforms.js (hunk 2 above)
Comment (kishorepmr) [reply packages/node_modules/@ciscospark/internal-plugin-conversation/test/unit/spec/decrypt-transforms.js]: Got it

**Pull request merged**